static/js/textMining.js

```diff
@@ -608,6 +607,7 @@ function drawAll(error, dataset) {
       }
       currentNode.selected = (!currentNode.holder)? true : false;
     } else {
+      setBanner({});
```

Comment:
this was just an accumulation of incremental changes that eventually rendered the setBanner function useless, just didn't realize. I took it out!